azure-pipelines-pr.yml

```diff
@@ -0,0 +1,108 @@
+variables:
+  - ${{ if eq(variables['System.TeamProject'], 'public') }}:
+    - name: PoolProvider
+      value: NetCore-Svc-Public
+  - ${{ if ne(variables['System.TeamProject'], 'public') }}:
+    - name: PoolProvider
+      value: NetCore1ESPool-Svc-Internal
+
+trigger: none
+
+pr:
+  branches:
+    include:
+    - '*'
+
+resources:
+  containers:
+  - container: LinuxContainer
+    image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-14.04-cross-0cd4667-20170319080304
+
+stages:
+- stage: build
+  displayName: Build
+  jobs:
+  - template: /eng/common/templates/jobs/jobs.yml
+    parameters:
+      enableMicrobuild: true
+      enablePublishBuildArtifacts: true
+      enablePublishBuildAssets: true
+      enablePublishUsingPipelines: true
+      enableTelemetry: true
+      helixRepo: dotnet/arcade-validation
+      jobs:
+      - job: Windows_NT
+        pool:
+          name: $(PoolProvider) # This is a queue-time parameter; Public default is NetCorePublic-Pool; Internal default is NetCoreInternal-Pool
+          demands: ImageOverride -equals windows.vs2019.amd64.open
```

Comment:
if there's an internal PR pipeline, this demand will fail. if there isn't such a pipeline, why is the `$(PoolProvider) conditional❔
same question for the other `pool` settings here